seed/static/seed/js/controllers/inventory_detail_controller.js

```diff
@@ -360,9 +360,10 @@ angular.module('BE.seed.controller.inventory_detail', [])
       $scope.diff = function () {
         if (_.isEmpty($scope.item_copy)) return {};
         // $scope.item_state, $scope.item_copy
+        const ignored_root_keys = ['extra_data', 'files', 'measures', 'scenarios']
         var result = {};
         _.forEach($scope.item_state, function (value, key) {
-          if (key === 'extra_data') return;
+          if (ignored_root_keys.includes(key)) return;
```

Comment:
Oh also, does this change anything practically speaking? or was this more of an enhancement (not building the key-values that we plan to just omit later)?

Reply:
This changes what data is returned in the diff. The only places this is used is for this functionality of updating states, so I don't think we have to worry about breaking anything.